geolib_plus/plot_settings.py

```diff
@@ -240,6 +247,7 @@ def assign_default_settings(self):
             "nbr_scale_units": 16,
             "font_size_labels": 8,
             "plot_size": self.__plot_sizes[0],
+            "distance_from_plot": -1,
```

Comment:
distance_from_plot is not a very clear name, maybe change it into something more descriptive like "distance_meta_data_from_plot"

Reply:
Ok!